routes/web.php

```diff
@@ -16,3 +16,12 @@
 Route::get('/airlines', function () {
     return view('airlines.index');
 });
+
+Route::get('/flights', function () {
+    return view('flights.index');
+});
+
+Route::get('/flights/{id}/edit', function ($id) {
+    return view('flights.edit', ['flightId' => $id]);
+});
```

Comment:
To make it easier in the frontend, you can do a controller and return the full flight instead of the id